### PR TITLE
[Update] Fix UI for user and admin

### PR DESCRIPTION
**Pull request by dinhthi12**
# Related Tickets

- [#Ticket_ID](https://edu-redmine.vn/issues/xxxx)

## WHAT this PR does?

- ex: Change number items `completed/total` in admin page.

## Notes Impacted Areas

- N/A.

## Attachments

- N/A.

### CODE REVIEW

File: app/javascript/stylesheets/_color.scss
Comment (hoangpt1310) [inline app/javascript/stylesheets/_color.scss:1]: variable nào dùng thì để lại không thì xóa

File: app/views/devise/confirmations/new.html.erb [hunk 1]
```diff
@@ -1,24 +1,58 @@
-<div class="card mb-3">
-  <div class="card-body">
-    <div class="pt-4 pb-2">
-      <h5 class="card-title text-center pb-0 fs-4"><%= t("pages.confirmations.title") %></h5>
-      <p class="text-center small"><%= t("pages.confirmations.text") %></p>
+<div class="
+card
+mb-3">
```
Comment (hoangpt1310) [inline app/views/devise/confirmations/new.html.erb]: xuống dòng

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -1,51 +1,122 @@
...
+    <div class="
+    pt-4
+    pb-2
+    ">
+      <h5 class="
+      card-title
+      text-center
+      pb-0
+      fs-4"
+      >
```
Comment (hoangpt1310) [inline app/views/devise/registrations/new.html.erb]: .

**Comment by tn0810520** [comment]
self check

**Comment by dinhthi12** [comment]
OKOK

**Pull request merged**